docs/api-reference/self-signed-certificates.md

```diff
@@ -0,0 +1,79 @@
+<!--pytest-codeblocks:skipfile-->
+<!-- markdownlint-disable-next-line first-line-h1 -->
+When using the `quilt3` API or CLI with a client-to-site (remote
+access) VPN you may need to use a self-signed certificate to
+securely connect. This requires adding the certificate to your
+Python environment.
+
+## Mac OS X
+
+1. Open a new browser window and navigate to any HTTPS website.
+2. Click the «lock icon» in the browser navigation/address bar.
+3. Click «View certificates» and record the certificate name value
+(e.g. `mycert`).
+4. Open Keychain Access.
+5. Select System Keychains > System Roots.
```

Comment:
If you use "1." for every item you don't have to renumber on edit. These bullets can be consolidated in any case.
```suggestion
1. Open Keychain Access and select System Keychains > System Roots
```